feat: test performances of Ollama on Raspberry Pi 5B

gmacario opened the pull request:
<!-- 
Thanks for creating this pull request 🤗

Please make sure that the pull request is limited to one type (docs, feature, etc.) and keep it as small as possible. You can open multiple prs instead of opening a huge one.
-->

## 📑 Description
<!-- Add a brief description of the pr -->

<!-- If this pull request closes an issue, please mention the issue number below -->
Closes #91 

<!-- You can also choose to add a list of changes and if they have been completed or not by using the markdown to-do list syntax
- [ ] Not Completed
- [x] Completed
-->

## ✅ Checks
<!-- Make sure your pr passes the CI checks and do check the following fields as needed - -->
- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information
<!-- Any additional information like breaking changes, dependencies added, screenshots, comparisons between new and old behavior, etc. -->

gmacario committed:
docs(howto): add howto-ollama-raspberry-pi-5b.md

gmacario committed:
feat: add code/open-webui

gmacario committed:
wip

gmacario committed:
wip: first prompt causes shutdown (possibly because of OOM?)

gmacario committed:
docs(ollama-on-rpi5b): comment about power supply

gmacario committed:
docs(ollama-on-rpi5b): successfully run qwen3:4b (CPU only)

gmacario committed:
chore: run 'prettier -w code/open-webui/compose.yml'

gmacario committed:
docs(ollama-on-rpi5b): gpt-oss:20b does not run with 8GB RAM

gmacario committed:
feat(ollama-on-rpi5b): text cleanup, add links, etc

gmacario committed:
fix(ollama-on-rpi5): update options to "ollama run"

OrsoEric reviewed (changes requested):
Add the performance on text, I'll make an additional table

gmacario committed:
docs(howto): update howto-install-ollama-raspberry-pi-5b.md

Adding text response produced by Ollama
as requested by https://github.com/B-AROL-O/FREISA/pull/92#pullrequestreview-3257514471

NOTE: Text generated by human-OCR from the above photo

OrsoEric reviewed (commented):
.

OrsoEric reviewed (commented):
.

OrsoEric reviewed (approved):
.

gmacario commented:
> .

Thank you @OrsoEric for your thorough and comprehensive review ❤️
